src/config/block.cpp

```diff
@@ -49,6 +49,13 @@ block::block(const block& other)
 {
 }
 
+block& block::operator=(const block& other)
+{
+    chain::block value = block(other);
+    value_ = std::move(value);
```

Comment:
I assume the move is required to be explicit here, as opposed to simply relying on the r-value?

Reply:
Exactly, I thought that like this it's more descriptive and it clearly shows that I'm forced to do a move assignment here. But your comment proves that this rather adds confusion than clarity. I'll change it such that it relies on the r-value.